utils/python/CIME/hist_utils.py

```diff
@@ -282,9 +301,23 @@ def generate_baseline(case, baseline_dir=None):
             if os.path.exists(baseline):
                 os.remove(baseline)
 
+            #special care for multi-instance cases,
+            #only keep first instance and
```

Comment:
I think that a multi-instance test with different results in each instance is a fundamentally different test than what we have now.   In the current tests we now allow for any number of ensemble members and expect them all to give the same result.   Putting all of them in the baseline dir is a waste of space.    This is also the logic by which the NCK test can compare the single instance output to the multi-instance output.